### ISSUE TITLE
New User Account for Alex P. with SDD (NN) (Waiting on user to log in)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#12050]
> What application are you using?

Other / Not Sure

> Describe the problem.

I do not have access to AMANDA. I field public inquiries into development cases and currently, I use the city's public platform to access case information. The public platform is an arduous method to collect and compare case information. I have to repeatedly input case numbers in the website search page to access case documents and information. This process takes a considerable amount of time and is redundant.

> How soon do you need this?

Urgently — Critical that this is addressed today

> Requested By
Alex P.

Request ID: DTS23-107175

**Comment by TracyLinder** [comment]
- Alex emailed for a status update.
- I sent the NURF to Alex.

**Comment by Nadin-Nader** [comment]
Emailed Alex to see if the NURF had been filled out. Once i get the NURF form i will be able to create the account

**Comment by Nadin-Nader** [comment]
Waiting for NURF:

> From: Phillips, Alex <Alex.Phillips@austintexas.gov> 
> Sent: Thursday, May 25, 2023 5:55 PM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>
> Subject: RE: Your DTS Service Request Has Been Submitted (#12050)
> 
> Thanks! I’ll have this ready next week!
>

**Comment by Nadin-Nader** [comment]
Created both AMANDA V7 TEST and PROD accounts. Waiting on response from user or seeing that he logged in.

**Comment by Nadin-Nader** [comment]
User has not responded but the account is active. If anything i will reopen or create a new card
